lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -10,16 +15,35 @@
 from rest_framework import status
 from rest_framework.test import APITestCase
 
+from course_modes.models import CourseMode
+from edx_rest_framework_extensions import config as edx_drf_config
+from lms.djangoapps.certificates.apis.v0.views import CertificatesDetailView
 from lms.djangoapps.certificates.models import CertificateStatuses
 from lms.djangoapps.certificates.tests.factories import GeneratedCertificateFactory
-from course_modes.models import CourseMode
+from openedx.core.djangoapps.waffle_utils import WaffleSwitch
+from openedx.core.lib.token_utils import JwtBuilder
 from student.tests.factories import UserFactory
 from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
 
 USER_PASSWORD = 'test'
 
 
+# TODO replace this with the one defined in OAuth Dispatch.
```

Comment:
It's a reminder to import this from oauth_dispatch rather than redefine it here - but only after Doug's PR merges.  Now that it has, I've updated this.